Resources/Prototypes/Reagents/narcotics.yml

```diff
@@ -314,6 +315,26 @@
   physicalDesc: reagent-physical-desc-acrid
   flavor: bitter
   color: "#215263"
+  metabolisms:
+    Narcotic:
+      effects:
+      - !type:MovespeedModifier
+        walkSpeedModifier: 0.65
+        sprintSpeedModifier: 0.65
```

Comment:
This doesn't seem to do anything, at least not until I vomited. Please test and check again.

Reply:
I've tested, and it works fine for me. The moment the impedrezene solution enters my spessman's chemstream, they slow down dramatically. Not sure why it didn't work for you. The speed reduction might be hard to feel over the other speed reduction caused by vomiting.

Reply:
That's good enough for me then.